### DISCUSSION TITLE
Visual Theme Editor

### DISCUSSION THREAD

**Discussion by Leo2426** [primefaces/community#651] [PrimeNG] [answered]
Hello, the theme editor is not available?
![Screenshot 2023-12-08 143230](https://github.com/primefaces/community/assets/93153931/381bf4dc-7b20-46c4-a71e-2bf9da7072ff)

**Comment by J-Marquis00** [comment]
Would be nice/useful if you can put it back online.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've removed the legacy visual theme editor since it's outdated and it's challenging to maintain. We've been working on a new theme designer for a while, we'll replace the legacy with the new one.

**Reply by dalenguyen** [reply]
Hi @cetincakiroglu & @cagataycivici will the domain name be changed in the new version? 

Existing domain: https://designer.primeng.org/
Legacy domain?: https://www.primefaces.org/designer-jsf/

I'm writing a book on PrimeNG, and I just want to make sure that the book reflects the Theme Designer correctly before publishing. This Visual Theme section will be reflected correctly. Could you please also provide some future insights from this feature? 

I can send you a sample section of the book for review if you prefer. My email is dale@dalenguyen.me. 

Thanks,

**Comment by LubosSremanak** [comment]
At what stage is the new editor? Is there any demo of the new one?

**Reply by timonmasberg** [reply]
https://github.com/orgs/primefaces/discussions/602, it is supposed to be available in Q1 next year. Let's see when it will actually be released, since it was pushed from Q2 this year to Q4 and now Q1..
